mmcv/runner/hooks/momentum_updater.py

```diff
@@ -404,11 +426,13 @@ class OneCycleMomentumUpdaterHook(MomentumUpdaterHook):
     """
 
     def __init__(self,
-                 base_momentum=0.85,
-                 max_momentum=0.95,
-                 pct_start=0.3,
-                 anneal_strategy='cos',
-                 three_phase=False,
+                 base_momentum: Union[float, List[float], Dict[str,
```

Comment:
It could be better to modify the docstring

Reply:
you mean to match the type hints?

Reply:
It seems the `dict` type of base_momentum is new.

Reply:
but from the implementation it is also expected to have dict as well